sql/server.go

```diff
@@ -165,10 +165,19 @@ func (s *Server) exec(req driver.Request) (driver.Response, error) {
 			return resp, err
 		}
 	}
+	if req.Txn != nil {
+		var err error
+		if planner.txn, err = client.UnmarshalTxn(*s.db, req.Txn); err != nil {
+			return resp, err
+		}
+	}
 	stmts, err := parser.Parse(req.Sql)
 	if err != nil {
 		return resp, err
 	}
+	if len(stmts) == 0 {
+		return resp, util.Errorf("sql: %s; unimplemented", req.Sql)
```

Comment:
The error message seems confusing. Perhaps something like `"sql: %s; contains no statements, nothing to do!"`.